composer.json

```diff
@@ -41,7 +41,15 @@
         "drupal/relaxed": "~1",
         "drupal/multiversion": "~1",
         "drupal/workspace": "~1",
-        "drupal/adobeanalytics": "dev-8.x-1.x"
+        "drupal/adobeanalytics": "dev-8.x-1.x",
```

Comment:
for this, composer.lock is better utilized, once we run an install, composer will create a composer.lock file with the exact versions of the downloaded properties.  Putting strict versions in composer.json is not a recommended approach.